File: tests/language_completion/fixtures/dart/boolean_break_continue.dart
void main() {
  print("dart_bool_loop");
  var total = 0;
  var ready = true;
  for (var i = 0; i < 6; i += 1) {
    if (i == 1) {
      continue;
    }
    if (i > 4) {
      break;
    }
    if (ready && i >= 2) {
      total += i;
    }
  }
  print(total);
}

File: tests/language_completion/fixtures/java/boolean_break_continue.java
class Main {
  static void main(String[] args) {
    System.out.println("java_bool_loop");
    int total = 0;
    boolean ready = true;
    for (int i = 0; i < 6; i = i + 1) {
      if (i == 1) {
        continue;
      }
      if (i > 4) {
        break;
      }
      if (ready && i >= 2) {
        total = total + i;
      }
    }
    System.out.println(total);
  }
}
